src/Database/IDatabase.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace Friendica\Database;
+
+interface IDatabase
```

Comment:
We already introduced the `I` for every other interface. Thus I don't want to remove it just at this place.

I know that Friendica has developers, who don't use full blown IDEs, so I think it would be better to explicit mark that this isn't a "normal" class